modules/mauth-authenticator-akka-http/src/test/scala/com/mdsol/mauth/akka/http/MAuthDirectivesSpec.scala

```diff
@@ -50,7 +50,8 @@ class MAuthDirectivesSpec extends WordSpec with Matchers with ScalatestRouteTest
   private val client = mock[ClientPublicKeyProvider]
   private val mockEpochTimeProvider: EpochTimeProvider = mock[EpochTimeProvider]
   private implicit val authenticator: RequestAuthenticator = new RequestAuthenticator(client, mockEpochTimeProvider)
-  private implicit val authenticatorV2: RequestAuthenticator = new RequestAuthenticator(client, mockEpochTimeProvider, true)
+  private val v2OnlyAuthenticate = true
+  private implicit val authenticatorV2: RequestAuthenticator = new RequestAuthenticator(client, mockEpochTimeProvider, v2OnlyAuthenticate)
```

Comment:
In principle this doesn't help you make sure that you're passing in the right parameters.

```
def abc(a: Boolean, b: Boolean) = ...

val a = true
val b = false
abc(b, a) // bug!
```

In Scala you have named parameters.

```
abc(a = true, b = false)
```

Reply:
btw this doesn't need to be an implicit (same for `authenticator`)

Reply:
updated as suggested https://github.com/mdsol/mauth-jvm-clients/pull/92/commits/07fa8b14f1df3b53cbccf169c0db9217561879bf